pkg/controller/che/che_controller.go

```diff
@@ -150,6 +151,49 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	var toRequestMapper handler.ToRequestsFunc = func(obj handler.MapObject) []reconcile.Request {
+		checlusters := &orgv1.CheClusterList{}
```

Comment:
I agree that having the logic of lines [155 to 166](https://github.com/eclipse/che-operator/pull/487/files#diff-d9445e614f0d728fd140771b0fd29c50R155-R166) extracted into a function like:

```go
func isTrustedBundleConfigMap(...) bool [}
```

might be nicer and prepare the way for the next step with will be using as many labelled trusted CA config maps as required.

Reply:
Done